app/shared/components/Wallet/Status/Actions.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import React, { Component } from 'react';
+import { Header, Grid, Loader, Segment, Visibility } from 'semantic-ui-react';
+import { translate } from 'react-i18next';
+
+import ActionsTable from './Actions/Table';
+
+type Props = {
+  actions: {
+    getAccountActions: () => void
+  },
+  actionObjects: {},
+  t: () => void,
+  validate: {}
+};
+
+class Actions extends Component<Props> {
+  props: Props;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      amount: 20
+    };
+  }
+
+  componentDidMount() {
+    this.tick();
+    this.interval = setInterval(this.tick.bind(this), 15000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  loadMore = () => this.setState({ amount: this.state.amount + 20 }, () => {
+    this.tick();
+  });
+
+  tick() {
+    const {
+      actions,
+      validate
+    } = this.props;
+    const {
+      getAccountActions
+    } = actions;
+
+    if (validate.NODE) {
+      const {
+        amount
+      } = this.state;
+
+      getAccountActions(amount);
```

Comment:
`getActions(account, start, end)` ideally is the format here.

Reply:
good idea!